### PR TITLE
Commerce 2255

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Comment by andrea-ale-sbarra** [comment]
@marco-leo , I'm sorry for being so long, but there were some commits to be fixed.. Hopefully this PR should be fine

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Comment by andrea-ale-sbarra** [comment]
ci:test

File: commerce-account-service/src/main/java/com/liferay/commerce/account/service/impl/CommerceAccountGroupCommerceAccountRelLocalServiceImpl.java [hunk 1]
```diff
@@ -54,6 +55,16 @@
...
 		}
 
+		CommerceAccountGroupCommerceAccountRel
+			commerceAccountGroupCommerceAccountRel2 =
+				commerceAccountGroupCommerceAccountRelPersistence.fetchByC_C(
+					commerceAccountGroupId, commerceAccountId);
+
+		if (commerceAccountGroupCommerceAccountRel2 != null) {
+			throw new DuplicateCommerceAccountGroupCommerceAccountRelException(
+				"Account already associated to AccountGroup ");
```
Comment (ethanbustad) [inline commerce-account-service/src/main/java/com/liferay/commerce/account/service/impl/CommerceAccountGroupCommerceAccountRelLocalServiceImpl.java:65]: Message should be `"Account already associated with AccountGroup"`, with no space at the end

File: commerce-account-service/src/main/java/com/liferay/commerce/account/service/impl/CommerceAccountLocalServiceImpl.java [hunk 2]
```diff
@@ -320,22 +320,23 @@ public CommerceAccount getGuestCommerceAccount(long companyId)
 
 		User defaultUser = userLocalService.getDefaultUser(companyId);
 
-		CommerceAccount commerceAccount = new CommerceAccountImpl();
+		CommerceAccountImpl commerceAccountImpl = new CommerceAccountImpl();
```
Comment (ethanbustad) [inline commerce-account-service/src/main/java/com/liferay/commerce/account/service/impl/CommerceAccountLocalServiceImpl.java:323]: Is this change necessary? It could just be a rebase error, or it could be a fix for some problem that I don't see. Let me know. If it's not necessary, please revert to the previous version.

File: headless/headless-commerce/headless-commerce-admin-account-impl/src/main/java/com/liferay/headless/commerce/admin/account/internal/resource/v1_0/AccountResourceImpl.java [hunk 3]
```diff
@@ -14,14 +14,27 @@
...
+import java.util.Map;
+
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ServiceScope;
+
```
Comment (ethanbustad) [inline headless/headless-commerce/headless-commerce-admin-account-impl/src/main/java/com/liferay/headless/commerce/admin/account/internal/resource/v1_0/AccountResourceImpl.java:29]: These imports should still be sorted below the `com.liferay.*` ones -- imports should be sorted entirely alphabetically.

**Review by ethanbustad — CHANGES_REQUESTED** [review]

File: commerce-account-service/src/main/java/com/liferay/commerce/account/service/impl/CommerceAccountLocalServiceImpl.java (hunk 2 above)
Comment (andrea-ale-sbarra) [reply commerce-account-service/src/main/java/com/liferay/commerce/account/service/impl/CommerceAccountLocalServiceImpl.java:323]: actually I had a strange conflict on this, as I didn't make anything on this, and also I resolved using "their" version, I'll just change this chunck to the previous version, ok?

**Comment by marco-leo** [comment]
sent to https://github.com/liferay/com-liferay-commerce/pull/1554
